docs/ref/models/constraints.txt

```diff
@@ -130,7 +130,7 @@ ensures the age field is never less than 18.
 ``UniqueConstraint``
 ====================
 
-.. class:: UniqueConstraint(*expressions, fields=(), name=None, condition=None, deferrable=None, include=None, opclasses=(), nulls_distinct=None, violation_error_code=None, violation_error_message=None)
```

Comment:
Thanks. I think we can avoid adding a release note for this signature change since as shown in your test and pointed out on the forum, we're just swapping one error for another.